cookie-banner-rules-list.json

```diff
@@ -2883,25 +2884,20 @@
     },
     {
       "click": {
-        "optIn": "button.dl_cookieBanner--buttonAll",
-        "optOut": "button.dl_cookieBanner--buttonSelected",
-        "presence": "div#dl_cookieBanner"
-      },
-      "cookies": {},
-      "id": "e3968548-ebca-4d6b-b8cf-5707e90fb612",
-      "domains": ["deepl.com"]
-    },
-    {
-      "click": {
-        "optIn": "button.js_cookieBannerPermissionButton",
-        "optOut": "span.js_cookieBannerProhibitionButton",
-        "presence": "section#cookieBanner"
+        "optIn": "button[data-testid=\"cookie-banner-strict-accept-all\"]",
+        "optOut": "button[data-testid=\"cookie-banner-strict-accept-selected\"]",
+        "presence": "div[data-testid=\"dl-cookieBanner\"]"
       },
       "cookies": {
-        "optOut": [{ "name": "cb", "value": "0" }]
+        "optOut": [
+          {
+            "name": "privacySettings",
+            "value": "%7B%22v%22%3A%221%22%2C%22t%22%3A111697155200%2C%22m%22%3A%22STRICT%22%2C%22consent%22%3A%5B%22NECESSARY%22%5D%7D"
+          }
+        ]
       },
-      "id": "c9adac7f-74db-4eab-9b97-114ff5954226",
-      "domains": ["otto.de"]
+      "id": "e3968548-ebca-4d6b-b8cf-5707e90fb612",
+      "domains": ["deepl.com"]
     },
```

Comment:
It's an arbitrary big value so that the banner doesn't get re-shown in a year or two. I updated it to be more "in bounds" so we don't break anything. The site seems to be fine with a timestamp in the future generally.

Reply:
The opt-in one is not really needed since the mechanism prioritizes opt-out and will only use it as a fallback.

Reply:
That should do the trick, yeah. Thank you for the information.